synapse/storage/databases/main/monthly_active_users.py

```diff
@@ -41,7 +41,14 @@ async def get_monthly_active_count(self) -> int:
         """
 
         def _count_users(txn):
-            sql = "SELECT COALESCE(count(*), 0) FROM monthly_active_users"
+            # Ensure we only fetch native users
```

Comment:
```suggestion
            # Exclude app service users
```
I wasn't quite sure what a 'native user' was.